src/sets.jl

```diff
@@ -435,6 +435,53 @@ Base.isapprox(a::T, b::T; kwargs...) where {T <: Union{SOS1, SOS2}} = isapprox(a
 
 dimension(s::Union{SOS1, SOS2}) = length(s.weights)
 
+"""
+	ActivationCond
+
+Activation condition for an indicator constraint.
+Used as first type parameter of `IndicatorSet{ActiveCond,S}`.
+"""
+@enum ActivationCond begin
+    ACTIVATE_ON_ZERO
+    ACTIVATE_ON_ONE
+end
+
+"""
+    IndicatorSet{A, S <: AbstractScalarSet}(set::S)
+
+``\\{((y, x) \\in \\{0, 1\\} \\times \\mathbb{R}^n : y = 0 \\implies x \\in set\\}``
+when `A` is `ACTIVATE_ON_ZERO` and
+``\\{((y, x) \\in \\{0, 1\\} \\times \\mathbb{R}^n : y = 1 \\implies x \\in set\\}``
+when `A` is `ACTIVATE_ON_ONE`.
+
+`S` has to be a sub-type of `AbstractScalarSet`.
+`A` is one of the value of the `ActivationCond` enum.
+`IndicatorSet` is used with a `VectorAffineFunction` holding
+the indicator variable first.
+
+Example: ``\\{(y, x) \\in \\{0, 1\\} \\times \\mathbb{R}^2 : y = 1 \\implies x_1 + x_2 \\leq 9 \\} ``
+
+```julia
+f = MOI.VectorAffineFunction(
+    [MOI.VectorAffineTerm(1, MOI.ScalarAffineTerm(1.0, z)),
+     MOI.VectorAffineTerm(2, MOI.ScalarAffineTerm(0.2, x1)),
+     MOI.VectorAffineTerm(2, MOI.ScalarAffineTerm(1.0, x2)),
+    ],
+    [0., 0.],
+)
+
+indicator_set = MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE}(MOI.LessThan(9.))
```

Comment:
9. should be 9.0 according to the JuMP style guide

Reply:
fixed, in tests also